spec/support/medium_controller_spec.rb

```diff
@@ -0,0 +1,79 @@
+RSpec.shared_examples "a medium controller" do
+  describe "GET 'index'" do
+    it "is successful" do
+      get :index
+      expect(response.status).to eq 200
+      expect(subject).to render_template :index
+    end
+  end
+
+  describe "GET 'new'" do
+    it "is successful" do
+      get :new
+      expect(response.status).to eq 200
+      expect(subject).to render_template :new
+    end
+  end
+
+  describe "GET 'show/:id'" do
+    it "renders show view" do
+      get :show, id: medium.id
+      expect(response.status).to eq 200
```

Comment:
As with the previous describe blocks, it might be worth it to `expect(subject).to render_template :show`.